Dockerfile

```
@@ -1,5 +1,5 @@
-FROM ckan/ckan-dev:2.9.10
-
+# Base stage for shared dependencies and extensions
+FROM ckan/ckan-base:2.10.4 as base
```

```suggestion
FROM ckan/ckan-base:2.10.4
```

Don't need the comment until #35 I guess.